src/integration-test/java/apptest/BusinessInformationIT.java

```diff
@@ -11,7 +11,6 @@
 // This suppresses the "(java:S2699)Tests should include assertions" rule.
```

Comment:
This comments is related to the removed annotation.

Reply:
Comment removed